### PR TITLE
Add first draft of expanded imCORE template

### PR DESCRIPTION
**Pull request by BrunoGrandePhD**
I've built on top of what @mattfazza did in #3. I tried my best to write the fully expanded version. I have two questions (for now) in comments starting with `# Question:` that could use your thoughts. Thanks!

### CODE REVIEW

File: templates/imcore.yaml [hunk 1]
```diff
@@ -0,0 +1,122 @@
+# Projects ----------------------------------------------------------
+
+imCORE Data Repository:
+  type: Project
+  acl:
+    # Question: Can you use the 'Ref' function in YAML keys?
```
Comment (zaro0508) [inline templates/imcore.yaml:6]: `!Ref` is not something that's built into standard yaml.  I believe the way to make it available is to create a custom yaml tag, https://stackoverflow.com/questions/43058050/creating-custom-tag-in-pyyaml

File: templates/imcore.yaml [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+            Study-032-Clinical:
+              type: Folder
+              children:
+                Study-032-Clinical.intake.yaml:
+                  type: File
+                  URI: file:///path/to/Study-032-Clinical.intake.yaml
+                Staging:
+                  type: Folder
+                  acl:
+                    imCORE Admins: edit & delete
```
Comment (thomasyu888) [inline templates/imcore.yaml]:
```suggestion
                    !Ref imCORE Admins: edit & delete
```

File: templates/imcore.yaml [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+            Study-032-WES:
+              type: Folder
+              children:
+                Study-032-WES.intake.yaml:
+                  type: File
+                  URI: file:///path/to/Study-032-WES.intake.yaml
+                Staging:
+                  type: Folder
+                  acl: 
+                    imCORE Admins: edit & delete
```
Comment (thomasyu888) [inline templates/imcore.yaml]:
```suggestion
                    !Ref imCORE Admins: edit & delete
```

File: templates/imcore.yaml [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+                    Level1:
+                      type: Folder
+                    Level2:
+                      type: Folder
+                    Level3:
+                      type: Folder
+                Staging:
+                  type: Folder
+                  acl: 
+                    imCORE Admins: edit & delete
```
Comment (thomasyu888) [inline templates/imcore.yaml]:
```suggestion
                    !Ref imCORE Admins: edit & delete
```

**Review by thomasyu888 — APPROVED** [review]

**Comment by thomasyu888** [comment]
LGTM as a first draft. If no objections, I will merge this @BrunoGrandePhD .  This way we can keep track of all the drafts that we come up with.

**Comment by BrunoGrandePhD** [comment]
Go ahead!

**Review by thomasyu888 — APPROVED** [review]

**Pull request merged**